modules/claims_api/app/controllers/concerns/claims_api/v2/disability_compensation_validation.rb

```diff
@@ -244,6 +245,7 @@ def validate_form_526_disability_secondary_disability_classification_code_matche
         )
       end
 
+      # TO-DO:Fix the dates here to handle the new format
```

Comment:
just confirming this TODO is intended for later

Reply:
Thanks for the heads up!